kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -133,6 +133,16 @@ class BatchJobSubmission(session: KyuubiBatchSessionImpl, batchRequest: BatchReq
     }
   }
 
+  def getOperationLogRowSet(
+      order: FetchOrientation,
```

Comment:
the order is never used

Reply:
yea.. I just leave it here since the thrift code path did so